source/deimos/openssl/ssl.di

```diff
@@ -1582,8 +1582,8 @@ enum SSL_VERIFY_PEER = 0x01;
 enum SSL_VERIFY_FAIL_IF_NO_PEER_CERT = 0x02;
 enum SSL_VERIFY_CLIENT_ONCE = 0x04;
 
-alias SSL_library_init OpenSSL_add_ssl_algorithms;
-alias SSL_library_init SSLeay_add_ssl_algorithms;
+auto OpenSSL_add_ssl_algorithms()() { return SSL_library_init(); }
+auto SSLeay_add_ssl_algorithms()() { return SSL_library_init(); }
```

Comment:
Why ? `alias` is better than a template

Reply:
Well, because the C code is
```c
# define OpenSSL_add_ssl_algorithms()    SSL_library_init()
```
and not
```c
# define OpenSSL_add_ssl_algorithms    SSL_library_init
```
.